lib/gds_zendesk/test_helpers.rb

```diff
@@ -55,7 +55,7 @@ def zendesk_returns_redirect
     end
 
     def zendesk_endpoint
-      "https://#{valid_zendesk_credentials["username"]}:#{valid_zendesk_credentials["password"]}@govuk.zendesk.com/api/v2"
```

Comment:
How come we removed the user/pass?

Reply:
User/Pass are no longer passed as part of the URL when stubbing, and now need to be required as a `with(basic_auth: ...)`